java.md

```diff
@@ -178,11 +178,52 @@ to manage each platform specific Jar.  Release individual architecture specific
 instance to download these released jars and package them into a combined Jar with a final `mvn release:perform`.
 An example of this methd can be found in the [Leveldbjni-native](https://github.com/fusesource/leveldbjni) `pom.xml` files. 
 
+### Remove Anti-patterns
+
+Anti-patterns can impact the performance on any instance family, but the level of impact can be different between instance families.  Below is a list of anti-patterns
+we have observed to impact Graviton positively when removed as much as possible:
```

Comment:
"we have observed to impact Graviton positively when removed" keeps flipping around positive/negative. I had to read it a second time to make fully parse it.

I recommend something like "Below is a list of anti-patterns we have found to be particularly impactful on Graviton:"

Reply:
Fixed.